ironfish/src/wallet/wallet.ts

```diff
@@ -680,14 +676,12 @@ export class Wallet {
       }
     }
 
-    const raw = await this.createTransaction(account, [], [mintData], [], {
+    return this.createTransaction(account, [], [mintData], [], {
```

Comment:
This function is analogous to send, so it should still post the transaction. If you don't want this use createRawTransaction. The design of this API is such that

 - createTransaction
 - postTransaction are the underlying mechanisms

and then there are 3 wrappers for people using the CLI for the base default cases
 - Wallet.send
 - Wallet.mint
 - Wallet.burn

Reply:
There is no `createRawTransaction()` in wallet. In wallet, `createTransaction()` is creating a raw transaction.
You slacked me to port Send UX flow to mint and burn.  The UX flow is a two step: it needs raw transaction to confirm and then post.
I can not use createTransaction api because there is extra logic to handle mint.

Reply:
Yes, the goal was to port the wallet:send UX flow to mint and burn. Where do we show the fee selector using inquirer like wallet:send? This is missing some of the UX flow.

What I'm getting at is that if you port the flow, you would know not to make changes to the mint RPC and command, because you'll just create 3 raw transactions just like wallet:send.